src/web/collectors/CollectorsStatusPage.jsx

```diff
@@ -96,12 +99,24 @@ const CollectorsStatusPage = createReactClass({
     }
   },
 
+  _importFinished() {
+    this.importsHelperModal.open();
+  },
+
   _formatStatus(name, item) {
-    let restart = null;
+    let buttons = null;
     if (name !== 'Status' && this.state.collector) {
-      restart = (<div className="pull-right">
-        <CollectorsRestartButton collector={this.state.collector} backend={name}/>
-      </div>);
+      if (Semver.gte(this.state.collector.collector_version, '0.1.8')) {
+        buttons = (<div className="pull-right">
+          <CollectorsImportButton collector={this.state.collector} backend={name} onFinish={this._importFinished}/>
+          &nbsp;
+          <CollectorsRestartButton collector={this.state.collector} backend={name}/>
+        </div>);
+      } else {
+        buttons = (<div className="pull-right">
+          <CollectorsRestartButton collector={this.state.collector} backend={name}/>
```

Comment:
Thanks. How about adding a disabled button with the info?
```suggestion
+          <Button bsStyle="info" bsSize="xsmall" disabled>Needs version &gt;= 0.1.8 to perform import</Button>
+          &nbsp;
          <CollectorsRestartButton collector={this.state.collector} backend={name}/>
```

Reply:
I am more into: If it's not working don't clutter the UI with non functional components.
We could also add a info text in the page header, not sure.
@edmundoa your call!?

Reply:
Those buttons are already in an odd place, so I think keeping it clean (and not adding the button if we can't perform the action) is better. Also, showing the requirement information inside a button doesn't really work imho.

Reply:
> I am more into: If it's not working don't clutter the UI with non functional components.

Which on the other hands, gives the user zero information on why the button is missing ;)